index.html

```diff
@@ -332,7 +454,7 @@ <h2>
         </dt>
         <dd>
           <p>
-            Initiates an exit from pointer lock state if currently locked to a
+            Initiates an exit from <a>pointer lock state</a> if currently locked to a
```

Comment:
We should turn this into an algorithm in the same way. It looks like it'll have two steps, if you take the above advice about centralizing the definition of "exit the pointer lock state":

1. If shadow-including root test fails, return.
1. Queue a task to exit pointer lock.

Then you'd move the text about the mouse cursor position into the "exit pointer lock" definition.

Reply:
Looks pretty good, but now you can just delete all the non-step paragraphs.